### PR TITLE
Feature/add cv tips

### PR DESCRIPTION
**Pull request by mjki4hj3**
Small updates to CV tips and cv tips are now closed on page load

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,28 +27,28 @@ Answer: We are ...
...
 <br>
-<br> _**Details go here**_
+
 
 --- 
 
 ### **How to write points for your work experience section:**
 
-- **Specific** - Mention a very particular situation/case/ticket/project you worked on: "Configured Multifactor Authentication for 10000+ employees"
+- **Specific** - Mention a very particular situation/case/ticket/project you worked on: "Configured multifactor authentication for 10,000+ employees"
```
Comment (adilyuken) [inline README.md]: MFA is a technology so using caps on the M and A seemed fine to me. What's your thoughts? Bad example I used here of course lmao, but maybe we can change the example so people don't get confused and make every starting letter a capital one.

File: README.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ Some examples:
 
 That is all you need. Recruiters and hiring managers are lazy and must go through a bunch of CVs. Get straight to the point and mention everything without a wasted word or letter. No one needs a heartbreaking story. Also, you will save space for points in later stages of your CV.
 
-Rule of 3 is a great writing technique you may remember from English classes. Only include buzzwords from jobs you're looking to apply to. 
+Rule of 3 is a great writing technique you may remember from english classes. Only include buzzwords from jobs you're looking to apply to.
```
Comment (adilyuken) [inline README.md]: English needs a capital E

**Review by adilyuken — COMMENTED** [review]
Added comments, see inline to code

File: README.md (hunk 1 above)
Comment (mjki4hj3) [reply README.md]: Ah yes of course, makes sense it just looked a bit awkward to me

File: README.md (hunk 2 above)
Comment (mjki4hj3) [reply README.md]: looks like i need some English classes

**Review by moabukar — APPROVED** [review]
lgtm - just implement those suggested fixes

**Review by a-abukar — COMMENTED** [review]
LGTM!!

**Pull request merged**